_events/4.markdown

```diff
@@ -1,15 +1,28 @@
 ---
 layout: event
 title:  "2019-10-08 Call"
-date:   2019-10-08 9:00
+date:   2019-10-08 10:00
 file_list:
-  # google drive links
-summary: This is a summary of the call.
+- (https://docs.google.com/presentation/d/1toZITnCCHTS7yoqODjop6qtI17CHCiyEJLInREQJSsc/edit#slide=id.g62e11bdf07_0_0)
```

Comment:
I think you need to remove the parentheses and also the dash.  Additionally, you need a description (like in the _example_templates) folder of each link.